### PR TITLE
Add secondary ebs volume setup in easy-setup.sh

### PR DESCRIPTION
**Pull request by KeitaW**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: 1.architectures/5.sagemaker-hyperpod/easy-setup.sh [hunk 1]
```diff
@@ -312,6 +312,13 @@ cat > cluster-config.json << EOL
       {
         "InstanceGroupName": "controller-machine",
         "InstanceType": "ml.m5.12xlarge",
+        "InstanceStorageConfigs": [
+          {
+            "EbsVolumeConfig": {
+              "VolumeSizeInGB": 500
```
Comment (mhuguesaws) [inline 1.architectures/5.sagemaker-hyperpod/easy-setup.sh:318]: Should be a parameter.

**Review by mhuguesaws — COMMENTED** [review]
left comments

File: 1.architectures/5.sagemaker-hyperpod/easy-setup.sh (hunk 1 above)
Comment (KeitaW) [reply 1.architectures/5.sagemaker-hyperpod/easy-setup.sh]: Makes sense. Addressed in https://github.com/aws-samples/awsome-distributed-training/pull/369/files#diff-eae4669f4ef447c7a71474ff7aebb3b1233cf545696670d6a54ff2e533e6b753

Comment (KeitaW) [reply 1.architectures/5.sagemaker-hyperpod/easy-setup.sh]: https://github.com/aws-samples/awsome-distributed-training/pull/369/commits/c8cf6248587293aa7442915c3149687523e1b9d3

**Review by mhuguesaws — APPROVED** [review]
LGTM

**Pull request merged**